.ruff.toml

```diff
@@ -0,0 +1,32 @@
+select = [
+    "E",    # pycodestyle
+    "W",    # pycodestyle
+    "F",    # pyflake
+    "I",    # isort
+    "B",    # flake8-bugbear
```

Comment:
Thanks for including comments for each of the rule categories!

Reply:
I saw this in some other repo and found it such a great idea as well.